tests/test_utils.py

```diff
@@ -271,27 +270,31 @@ class PlainRelatedResourceTypeSerializer(serializers.Serializer):
     assert get_related_resource_type(field) == output
 
 
-def test_get_included_serializers():
```

Comment:
as `get_included_serializers` should be deprecated instead of being removed this test should remain, check that a deprecation warning has been raised and test as before. This way we can be sure that the code is backwards compatible.